omnibus_overrides.rb

```diff
@@ -1,11 +1,6 @@
 # THIS IS NOW HAND MANAGED, JUST EDIT THE THING
-# .travis.yml and appveyor.yml consume this,
-# try to keep it machine-parsable.
+# .travis.yml and appveyor.yml consume this, try to keep it machine-parsable.
```

Comment:
This comment is no longer valid. We used it previously to install bundler / rubygems

Reply:
technically its not a bad thing to keep it machine-parseable though.

Reply:
I meant just the bit about travis / appveyor

Reply:
I removed the whole line. I was going to say "Please try to keep it machine-parsable" but then the obvious question was "for what? how do I know if the machine can parse this or not" and it isn't really doing anyone favors.